### ISSUE TITLE
Quick fix removes latex commands

### ISSUE THREAD

**Issue by t-huyeng** [valentjn/vscode-ltex#349]
Hey,

thanks for this great extension. I could not finde an issue that fits my problem. 

**Describe the bug**
Quick fix is replacing other words or commands instead of only the wrong word. This happens for example to me with items or sections. 

**Steps to reproduce**

1. Create an itemize environment
2. write something wrong 
3. Click on "Quick Fix"
4. Choose a correct word
5. See bug that not only the word but also /item gets replaced

**Expected behavior**
Only the wrong word should be replaced 

**Sample document**

<details>

```
\documentclass{article}
    % General document formatting

\begin{document}


\begin{itemize}
\item test % this should be replaced with Test - but the quick fix is replacing "\item test" with "Test"
\end{itemize}

\end{document}
```

</details>

**LTeX configuration**

<details>

```
"ltex.language": "de-DE",
```

</details>

**"LTeX Language Server" log file**

<details>

```
Juli 06, 2021 7:51:34 NACHM. org.bsplines.ltexls.server.DocumentChecker checkAnnotatedTextFragment
FEIN: Obtained 0 rule matches
```

</details>

**"LTeX Language Client" log file**

<details>

```
[Trace - 19:53:33] Sending request 'textDocument/codeAction - (55)'.
Params: {
    "textDocument": {
        "uri": "file:..../document.tex"
    },
    "range": {
        "start": {
            "line": 7,
            "character": 10
        },
        "end": {
            "line": 7,
            "character": 10
        }
    },
    "context": {
        "diagnostics": [
            {
                "range": {
                    "start": {
                        "line": 6,
                        "character": 0
                    },
                    "end": {
                        "line": 7,
                        "character": 10
                    }
                },
                "message": "Dieser Satz fängt nicht mit einem großgeschriebenen Wort an. – UPPERCASE_SENTENCE_START",
                "severity": 3,
                "source": "LTeX"
            }
        ]
    }
}


[Trace - 19:53:33] Received response 'textDocument/codeAction - (55)' in 2ms.
Result: [
    {
        "title": "Use 'Test'",
        "kind": "quickfix.ltex.acceptSuggestions",
        "diagnostics": [
            {
                "range": {
                    "start": {
                        "line": 6,
                        "character": 0
                    },
                    "end": {
                        "line": 7,
                        "character": 10
                    }
                },
                "severity": 3,
                "source": "LTeX",
                "message": "Dieser Satz fängt nicht mit einem großgeschriebenen Wort an. – UPPERCASE_SENTENCE_START"
            }
        ],
        "edit": {
            "documentChanges": [
                {
                    "textDocument": {
                        "version": 83,
                        "uri": "file:..../document.tex"
                    },
                    "edits": [
                        {
                            "range": {
                                "start": {
                                    "line": 6,
                                    "character": 0
                                },
                                "end": {
                                    "line": 7,
                                    "character": 10
                                }
                            },
                            "newText": "Test"
                        }
                    ]
                }
            ]
        }
    },
    {
        "title": "Hide false positive",
        "kind": "quickfix.ltex.hideFalsePositives",
        "diagnostics": [
            {
                "range": {
                    "start": {
                        "line": 6,
                        "character": 0
                    },
                    "end": {
                        "line": 7,
                        "character": 10
                    }
                },
                "severity": 3,
                "source": "LTeX",
                "message": "Dieser Satz fängt nicht mit einem großgeschriebenen Wort an. – UPPERCASE_SENTENCE_START"
            }
        ],
        "command": {
            "title": "Hide false positive",
            "command": "_ltex.hideFalsePositives",
            "arguments": [
                {
                    "uri": "...../document.tex",
                    "falsePositives": {
                        "de-DE": [
                            "{\"rule\":\"UPPERCASE_SENTENCE_START\",\"sentence\":\"^\\\\Qtest\\\\E$\"}"
                        ]
                    }
                }
            ]
        }
    },
    {
        "title": "Disable rule",
        "kind": "quickfix.ltex.disableRules",
        "diagnostics": [
            {
                "range": {
                    "start": {
                        "line": 6,
                        "character": 0
                    },
                    "end": {
                        "line": 7,
                        "character": 10
                    }
                },
                "severity": 3,
                "source": "LTeX",
                "message": "Dieser Satz fängt nicht mit einem großgeschriebenen Wort an. – UPPERCASE_SENTENCE_START"
            }
        ],
        "command": {
            "title": "Disable rule",
            "command": "_ltex.disableRules",
            "arguments": [
                {
                    "uri": "file:..../document.tex",
                    "ruleIds": {
                        "de-DE": [
                            "UPPERCASE_SENTENCE_START"
                        ]
                    }
                }
            ]
        }
    }
]
```

</details>

**Version information**
List here the version information of the relevant software.

- Windows_NT x64 10.0.19042
- VS Code: 1.57,1
- vscode-ltex: 10.4.0

Small gif: https://imgur.com/a/LqcOzae

**Comment by hpvd** [comment]
same here using v10.4.0 on 1.57.1

**Comment by valentjn** [comment]
I think this got introduced in valentjn/ltex-ls#69. It should be fixed now.

**Comment by valentjn** [comment]
Fix released in 10.5.0.

**Comment by t-huyeng** [comment]
Thanks @valentjn for the fast fix, with version 10.5.0. it works for me.
